src/execution_algorithms/ltfb/random_pairwise_exchange.cpp

```diff
@@ -245,11 +314,11 @@ void RandomPairwiseExchange::select_next(model& m,
                            " (trainer ",
                            local_trainer,
                            " score = ",
-                           local_score,
+                           local_scores.begin()->second,
```

Comment:
Might be helpful (informative) to print score for each metrics and not just the first metric.

Reply:
Yeah, we don't have a canoncal `operator<<(std::ostream&, std::vector<T> const&)` overload. This was just for simplicity. I'll expand it.